DEVELOPMENT.md

```diff
@@ -101,41 +101,41 @@
 
 ## Test Running
 
--   Run all unit tests.
+-   Run all unit and integration tests.
 
     ```sh
-    bazel test //...
+    bazel test ...
     ```
 
--   Run a single unit test like `JavaCodeGeneratorTest.java`
+-   Run all unit tests.
 
     ```sh
-    bazel run //src/test/java/com/google/api/generator/engine:JavaCodeGeneratorTest
+    bazel test $(bazel query ... | grep ^//:unit_)
     ```
 
--   Run a single integration test for API like `Redis`, it generates Java source
-    code using the Java microgenerator and compares them with the goldens files
-    in `test/integration/goldens/redis`.
+-   Run a single unit test like `JavaCodeGeneratorTest.java`
 
     ```sh
-    bazel test //test/integration:redis
+    bazel run :unit_com_google_api_generator_engine_JavaCodeGeneratorTest
     ```
 
--   Run all unit and integration tests.
+-   Update unit test golden files, for example `JavaCodeGeneratorTest.java`:
 
     ```sh
-    bazel test $(bazel query "src/test/..." | grep "Test$") //test/integration/...
+    bazel run :update_com_google_api_generator_engine_JavaCodeGeneratorTest
     ```
 
--   Update unit test golden files, for example `JavaCodeGeneratorTest.java`:
+-   Run a single integration test for API like `Redis`, it generates Java source
+    code using the Java microgenerator and compares them with the goldens files
+    in `test/integration/goldens/redis`.
 
     ```sh
-    bazel run //src/test/java/com/google/api/generator/engine:JavaCodeGeneratorTest_update
+    bazel test test/integration:redis
```

Comment:
This is somewhat subjective, but please put `//` back in documentation (in all places). This makes it a bit more robust because it unequivocally describes the targets.

Reply:
done.